Main/ClientApp/src/Components/Drawer/DrawerItemNested.tsx

```diff
@@ -55,7 +55,16 @@ export default function DrawerItemNested(props: DrawerItemNestedProps) {
           <MoreVert />
         </IconButton>
       </ListItem>
-      <Menu open={anchorEl !== null} anchorEl={anchorEl} onClose={onMenuClose}>
+      <Menu
+        open={anchorEl !== null}
+        anchorEl={anchorEl}
+        onClose={onMenuClose}
+        PaperProps={{
+          style: {
+            maxHeight: 350,
+          },
```

Comment:
Is this a fix for the title and icon not being in the same line? If so, it should not be done like that as it is not responsive and will probably break when the title is longer.

Reply:
in the mockup we have contributors to collection below settings, therefore i set the limit for height of menu -> if there are more contributors menu will have scroll

Reply:
this is not fix for listitem, this still needs to be fixed